### PR TITLE
Bug#1444976 - fixes - added descriptions for openshift_image_tag and openshift_pkg_verison

### PR DESCRIPTION
**Pull request by gaurav-nelson**
For https://bugzilla.redhat.com/show_bug.cgi?id=1444976

- Added missing variables and associated warnings to the docs.

### CODE REVIEW

**Comment by gaurav-nelson** [comment]
@openshift/team-documentation PTAL 🛩

File: install_config/install/advanced_install.adoc [hunk 1]
```diff
@@ -390,8 +390,23 @@ details.
...
+|Use this variable to specify a container image tag to install or configure.
+
+|`openshift_pkg_version`
+|Use this variable to specify a rpm version to install or configure.
+
 |===
 
+[WARNING]
+====
+Be careful with modifying the `openshift_image_tag` or the `openshift_pkg_version` variables after the cluster is set up, as it could trigger an upgrade and result in downtime.
```
Comment (ahardin-rh) [inline install_config/install/advanced_install.adoc]: Maybe an if, then statement here?
If you modify the `openshift_image_tag` or the `openshift_pkg_version` variables after the cluster is set up, then an upgrade can be triggered, resulting in downtime.

File: install_config/install/advanced_install.adoc [hunk 2]
```diff
@@ -390,8 +390,23 @@ details.
...
+
+* If `openshift_image_tag` is set, its value is used for all hosts in containerized environments, even those that have another version installed.
+* If `openshift_pkg_version` is set, its value is used for all hosts in RPM based environments, even those that have another version installed.
```
Comment (ahardin-rh) [inline install_config/install/advanced_install.adoc]: RPM-based environments

**Comment by gaurav-nelson** [comment]
@ahardin-rh Thank you, fixed those nits!

**Comment by gaurav-nelson** [comment]
Minor changes no revision history needed.

File: install_config/install/advanced_install.adoc [hunk 3]
```diff
@@ -390,8 +390,27 @@ details.
...
 
+[WARNING]
+====
+If you modify the `openshift_image_tag` or the `openshift_pkg_version` variables
+after the cluster is set up, then an upgrade can be triggered, resulting in
+downtime.
+
+* If `openshift_image_tag` is set, its value is used for all hosts in
+containerized environments, even those that have another version installed. If
+* `openshift_pkg_version` is set, its value is used for all hosts in RPM-based
```
Comment (wmengRH) [inline install_config/install/advanced_install.adoc]: I am not sure "If" should be in the same line of the sentence or in another line on purpose.
@gaurav-nelson

Comment (gaurav-nelson) [reply install_config/install/advanced_install.adoc]: Thanks @wmengRH 
Its not on purpose, I'll fix it soon.

**Pull request merged**